sw/device/tests/autogen/alert_test.c

```diff
@@ -550,19 +551,21 @@ static void trigger_alert_test(void) {
         &alert_handler, exp_alert));
   }
 
-  // Write otp_ctrl's alert_test reg and check alert_cause.
-  for (dif_otp_ctrl_alert_t i = 0; i < 5; ++i) {
-    CHECK_DIF_OK(dif_otp_ctrl_alert_force(&otp_ctrl, kDifOtpCtrlAlertFatalMacroError + i));
+  if (kBootStage != kBootStageOwner) {
```

Comment:
maybe good to add a comment saying that this is due to the ePMP config in the ROM_EXT. we can remove this once a more granular option is available #[20348](https://github.com/lowRISC/opentitan/issues/20348)

Reply:
Done.